src/openrct2/world/Location.hpp

```diff
@@ -810,9 +808,14 @@ struct MapRange : public RectRange<CoordsXY>
 
     constexpr MapRange Normalise() const
     {
+        // Don't use std::min/max, as they require <algorithm>, one of C++'s heaviest
+        // in this very common header.
         auto result = MapRange(
-            std::min(GetLeft(), GetRight()), std::min(GetTop(), GetBottom()), std::max(GetLeft(), GetRight()),
-            std::max(GetTop(), GetBottom()));
+            GetLeft() > GetRight() ? GetRight() : GetLeft(), // min
+            GetTop() > GetBottom() ? GetBottom() : GetTop(), // min
```

Comment:
For clarity, could I suggest going with:
```suggestion
            GetLeft() < GetRight() ? GetLeft() : GetRight(), // min
            GetTop() < GetBottom() ? GetTop() : GetBottom(), // min
```